timeless/companies/models.py

```diff
@@ -25,5 +25,7 @@ class Company(TimestampsMixin, DB.Model):
     items = DB.relationship("Item", order_by="Item.id",
                             back_populates="company")
 
+    @validate_required("name", "code")
```

Comment:
@marceloamadeu `@` means it's a decorator which decorates a method following in the next line (so plz remove blank line you've added after each decorator :)). So here it decorates `__repr__` and it does not make any sense. This decorator simply checks that arguments of decorated function contain provided to `validate_required` function args